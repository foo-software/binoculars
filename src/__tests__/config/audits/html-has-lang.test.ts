import auditHtmlHasLang from '../../../config/audits/html-has-lang';

describe('html-has-lang audit', () => {
  it('should match snapshot', () => {
    expect(auditHtmlHasLang('en')).toMatchSnapshot();
  });
});
