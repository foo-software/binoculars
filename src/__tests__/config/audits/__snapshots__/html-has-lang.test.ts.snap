// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`html-has-lang audit should match snapshot 1`] = `
Object {
  "description": "If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more](https://web.dev/html-has-lang/).",
  "title": "\`<html>\` element has a \`[lang]\` attribute",
}
`;
